pkg/convert/convert_test.go

```diff
@@ -0,0 +1,331 @@
+package convert
```

Comment:
Can we rename this package to `apkbuild` to make it consistent with python and gem?

Reply:
it is an apkbuild file; if anything we should probably change python to pypi ?